tests/integration/src/account.rs

```diff
@@ -0,0 +1,41 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Astar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Astar is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+use crate::setup::*;
+use std::str::FromStr;
+
+#[test]
+fn transfer_to_h160_via_lookup() {
+    new_test_ext().execute_with(|| {
+        let eth_address = H160::from_str("0xaaafB3972B05630fCceE866eC69CdADd9baC2771").unwrap();
```

Comment:
Given the experiences so far, maybe don't put concrete addresses (or private keys 🙈 ) in the code. 

It's safer practice to always just derive it - e.g. Alice or Bob.

On another note, it's also a _magic_ value - where did this come from?
Calling setup action, and then just reading the value out of storage would be better practice IMHO.

Reply:
The EVM Address is copied from dev chain's [genesis config and It was used in RPC tests too](https://github.com/search?q=repo:AstarNetwork/Astar%200xaaafB3972B05630fCceE866eC69CdADd9baC2771&type=code).

Reply:
Yeah, I think that very "place" for address caused someone to lose a lot of ASTR 🙈 

As good practice, I'd recommend to not do this, and instead just derive address.
That way you know 100% you won't accidentally copy/paste your private key (or address) into the code.

Reply:
I didn't knew that. 
Ok, I'll change the hardcoded address to derived one